### PR TITLE
Fix the CI

### PR DESCRIPTION
**Pull request by mithro**
* Update to the latest KLayout version
 * Don't pre-install the deps (as they get out of date), instead allow the `dpkg` command to fail and the `apt` command to fix the dependencies.

Fixes #29.

### CODE REVIEW

**Comment by proppy** [comment]
This currently fails with:
```
Info: Could not load the Qt platform plugin "xcb" in "" even though it was found.
Fatal: This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: eglfs, linuxfb, minimal, minimalegl, offscreen, vnc, xcb.
```

File: .github/workflows/Tests.yml [hunk 1]
```diff
@@ -53,6 +53,18 @@ jobs:
...
+        run: |
+          sudo apt update -qq -y
+          wget https://www.klayout.org/downloads/Ubuntu-22/klayout_0.28.2-1_amd64.deb
+          # The dpkg command will fail complaining about missing dependencies.
+          sudo dpkg -i ./klayout_0.28.2-1_amd64.deb || true
+          # The apt install command should install the missing dependencies
+          # needed by KLayout above and finish the install.
+          sudo apt install -f -y
+          # Check that KLayout was successfully installed!
+          klayout --version
```
Comment (proppy) [inline .github/workflows/Tests.yml]: This is what `klayout --version` returns:
```
(base) ~ 🍙 klayout --version
ERROR: Unknown option: --version
```
I suspect it's:
```
klayout -v
```

File: .github/workflows/Tests.yml [hunk 2]
```diff
@@ -53,6 +53,18 @@ jobs:
 
     steps:
 
+      - name: Installing Klayout
+        run: |
+          sudo apt update -qq -y
+          wget https://www.klayout.org/downloads/Ubuntu-22/klayout_0.28.2-1_amd64.deb
```
Comment (proppy) [inline .github/workflows/Tests.yml:59]: do we want to update to https://anaconda.org/LiteX-Hub/klayout to get klayout `HEAD`? do we want to stick to released point version?

Comment (mithro) [reply .github/workflows/Tests.yml:59]: This is the smallest change to get the CI green again. We can discuss using different KLayout version later.

File: .github/workflows/Tests.yml (hunk 1 above)
Comment (mithro) [reply .github/workflows/Tests.yml]: Tried `klayout --help` instead?

File: .github/workflows/Tests.yml [hunk 3]
```diff
@@ -57,8 +57,13 @@ jobs:
         run: |
           sudo apt update -qq -y
           wget https://www.klayout.org/downloads/Ubuntu-22/klayout_0.28.2-1_amd64.deb
+          # The dpkg command will fail complaining about missing dependencies.
           sudo dpkg -i ./klayout_0.28.2-1_amd64.deb || true
+          # The apt install command should install the missing dependencies
+          # needed by KLayout above and finish the install.
           sudo apt install -f -y
+          # Check that KLayout was successfully installed!
+          klayout --help
```
Comment (proppy) [inline .github/workflows/Tests.yml]: I don't think that's working either:
```
(base) ~ 🍡 klayout --help
ERROR: Unknown option: --help
```

Comment (proppy) [reply .github/workflows/Tests.yml]: but this should:
```
(base) ~ 👺 klayout -h
klayout [<options>] [<file>] ..
options
  -b                  Batch mode (same as -zz -nc -rx)
  -c <config file>    Use this configuration file
  -nc                 Don't use a configuration file (implies -t)
  -d <debug level>    Set debug level
  -e                  Editable mode (allow editing of files)
  -ne                 Readonly mode (editing of files is disabled)
  -gr <file name>     Record GUI test file
  -gp <file name>     Replay GUI test file
  -gb <line number>   Replay GUI test file up to (including) line
  -gx <millisec>      Replay rate for GUI test file
  -gi                 Incremental logs for GUI test file
  -i                  Disable undo buffering (less memory requirements)
  -ni                 Enable undo buffering (default, overrides previous -i option)
  -j <path>           Add the given path to the macro project paths
  -l <lyp file>       Use layer properties file
  -lx                 With -l: add other layers as well
  -lf                 With -l: use the lyp file as it is (no expansion to multiple layouts)
  -m <database file>  Load RDB (report database) file (into previous layout view)
  -mn <database file> Load L2NDB (layout to netlist database) file (into previous layout view)
  -n <technology>     Technology to use for next layout(s) on command line
  -nn <tech file>     Technology file (.lyt) to use for next layout(s) on command line
  -p <plugin>         Load the plugin (can be used multiple times)
  -r <script>         Execute main script on startup (after having loaded files etc.)
  -rr <script>        Like -r, but does not exit after executing the script
  -rm <script>        Execute script on startup before loading files (can be used multiple times)
  -rd <name>=<value>  Specify script variable
  -rx                 Ignore all implicit macros (*.rbm, rbainit, *.lym)
  -s                  Load files into same view
  -t                  Don't update the configuration file on exit
  -nt                 Update the configuration file on exit (default, overrides previous -t option)
  -u <file name>      Restore session from given file
  -v                  Print program version and exit
  -wd <name>=<value>  Define a variable within expressions
  -x                  Synchronous drawing mode
  -y <package>        Package installation: install package(s) and exit - can be used more than once
                      ('package' is a name, an URL and optionally a version in round brackets)
  -yd                 With -y: include dependencies
  -z                  Non-GUI mode (hidden views)
  -zz                 Non-GUI mode (database only, implies -nc)
```

File: .github/workflows/Tests.yml [hunk 4]
```diff
@@ -53,6 +53,28 @@ jobs:
...
+          sudo apt install -qq -y \
+            libqt5multimediawidgets5 \
+            libqt5svg5-dev \
+            libqt5xmlpatterns5-dev \
+            qt5-qmake \
+            qtbase5-dev \
+            qtbase5-dev-tools \
+            qtchooser \
+            qtmultimedia5-dev \
+            qttools5-dev \
```
Comment (proppy) [inline .github/workflows/Tests.yml]: I don't think you want the trailing `\` here.

File: .github/workflows/Tests.yml [hunk 5]
```diff
@@ -53,6 +53,28 @@ jobs:
 
     steps:
 
+      - name: Installing Klayout
+        run: |
+          sudo apt update -qq -y
+          sudo apt install -qq -y \
```
Comment (proppy) [inline .github/workflows/Tests.yml]: `apt-get -f install` should work as long as we use the right command `-v` or `-h` for testing (i.e: it shouldn't try to interact with the ui subsystem).

**Pull request merged**